specifyweb/frontend/js_src/lib/attachmentplugin.js

```diff
@@ -52,7 +54,10 @@ module.exports =  UIPlugin.extend({
                 .append(self.progressBar)
                 .dialog({ modal:true });
 
-            attachments.uploadFile(file, function(progressEvt) {
+            if (isPublicImage) { var attachmentserver = attachmentserverpublic;}
```

Comment:
It's a bit strange to declare a variable in the separate branches of an if-statement. More idiomatic is `var attachmentserver = isPublicImage ? attachmentserverpublic : attachmentserverprivate;`

Reply:
Great! I didn't know how to do that in JS.